http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/fault/ServerConnectivityErrorMessageTest.java

```diff
@@ -137,6 +160,20 @@ private Throwable captureException() {
         throw new AssertionError("Call did not fail as expected.");
     }
 
+    private Throwable captureExceptionWithHttpContinueResponse() {
```

Comment:
Nit: Consider adding the future as a parameter to `captureException` and keep it to one method